tests/tests/classes/Plugin.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
+
+/**
+ * Test: Plugin Function Class
+ */
+
+const expect = require('chai').expect;
+const Plugin = require('../../../lib/classes/Plugin')({});
+
+const SPlugin = new Plugin();
+
+describe('Plugin class', () => {
+
+  after((done) => {
+    done();
+  });
+
+  it('should have a name', () => {
+    const name = SPlugin.getName();
+
+    expect(name.length).to.be.at.least(1);
```

Comment:
Where did we set a name for this project? Looks like this is a default name? Not sure how this can pass if we don't set a name ourselves.